cedar-policy/src/api.rs

```diff
@@ -127,7 +129,7 @@ impl Entity {
             evaluator
                 .interpret(expr.as_borrowed())
                 .map(EvalResult::from)
-                .map_err(|e| EvaluationError::StringMessage(e.to_string())),
+                .map_err(EvaluationError::from),
```

Comment:
I think you can just drop the `map_err` here

Reply:
Dropped map_err.